transform/mattermost-analytics/models/staging/stripe/stg_stripe__invoice_line_items.sql

```diff
@@ -0,0 +1,38 @@
+with source as (
+
+    select * from {{ source('stripe_raw', 'invoice_line_items') }}
+
+),
+
+invoice_line_items as (
+
+    select
+          amount as amount
+        , currency as currency
+        , description as description
+        , discountable as discountable
+        , discounts as discounts
+        , discount_amounts as discount_amounts
+        , id as id
```

Comment:
```suggestion
        , id as invoice_line_item_id
```
Let's follow the naming conventions.